chart/kubeapps/templates/tiller-proxy-deployment.yaml

```diff
@@ -47,6 +47,7 @@ spec:
           args:
             - --host={{ .Values.tillerProxy.host }}
             - --user-agent-comment=kubeapps/{{ .Chart.AppVersion }}
+            - --kubeapps-namespace={{ .Release.Namespace }}
```

Comment:
this is not necessary, the namespace is available in the env var `POD_NAMESPACE` (defined a bit below)